css/02-site-header-and-nav.css

```diff
@@ -1,14 +1,77 @@
 /* --- temporary overwrite to get the nav out of the way --- */
-#spine{
-  display:none;
-}
 
 .spine-mobile #spine {
-  display: block;
+	display: block;
 }
 
 /* ------- */
-#site-header{
-  min-height:150px;
-  background: #fff;
+#site-header {
+	background: #fff;
+}
+@media (min-width: 989px) {
+  #wsu-signature{
+    position:absolute;
+    left:2em;
+    top:1em;
+    width:150px;
+    height:92px;
+  }
+	#spine {
+		min-height: auto!important;
```

Comment:
@slocker - Getting this warning from CSS Lint:

```
WARNING: Expected (<length> | <percentage> | <content-sizing> | contain-floats | -moz-contain-floats | -webkit-contain-floats) but found 'auto'. Properties should be known (listed in CSS3 specification) or be a vendor-prefixed property. (known-properties) Browsers: All
```

It looks like this used to be a thing, but is no longer a thing. https://developer.mozilla.org/en-US/docs/Web/CSS/min-height#Browser_compatibility

https://travis-ci.org/washingtonstateuniversity/medicine.wsu.edu/builds/165422358